src/components/heatmap/Heatmap.js

```diff
@@ -358,7 +364,7 @@ const Heatmap = forwardRef((props, deckRef) => {
   // Generate the axis label, axis title, and loading indicator text layers.
   const textLayers = [
     new HeatmapCompositeTextLayer({
-      showLoadingIndicator: backlog.length,
+      showLoadingIndicator: false,
```

Comment:
What is happening here?

Reply:
I switched from showing the text "Loading..." as a heatmap DeckGL TextLayer to using a callback to be able to use the spinning animation from the parent TitleInfo component. So I can actually delete the `showLoadingIndicator` code entirely.